utils.js

```diff
@@ -32,28 +34,32 @@ const isValidNetwork = (network) => {
     return true;
 };
 
-const signedNumberToHexStringLE = (number) => {
-
-    let numberAsHex = numberToHexString(number);
+/**
+ *
+ * @param {number} number to convert to a hex string representation in LE format
+ * @returns {string} returns the string hex representation of the number in LE format
```

Comment:
```suggestion
 * @returns {string} returns the string hex representation of the signed number in LE format. If the most significant bit equals 1 it will add an extra 0 byte
```
Feel free to reword it, the main idea is to make clear what the output will be